### PR TITLE
[doxy][addons] Sync doxygen documentation with code changes.

### PR DESCRIPTION
**Pull request by ace20022**

### CODE REVIEW

File: xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Peripheral.h [hunk 1]
```diff
@@ -674,8 +673,8 @@ namespace addon
     /*!
      * @brief Notify the frontend that button maps have changed
      *
-     * @param[optional] deviceName The name of the device to refresh, or empty/null for all devices
-     * @param[optional] controllerId The controller ID to refresh, or empty/null for all controllers
+     * @param[optional] strDeviceName The name of the device to refresh, or empty/null for all devices
+     * @param[optional] strControllerId The controller ID to refresh, or empty/null for all controllers
      */
     void RefreshButtonMaps(const std::string& strDeviceName = "", const std::string& strControllerId = "")
```
Comment (garbear) [inline xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Peripheral.h]: Can you actually go in the other direction, and remove the `str` prefixes here?

**Comment by garbear** [comment]
Looks good, only one comment.

File: xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Peripheral.h (hunk 1 above)
Comment (ace20022) [reply xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Peripheral.h]: Sure, it's nicer without the prefix.

**Comment by ace20022** [comment]
I know that I'm a minimalist ;) better now?

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Pull request merged**